### PR TITLE
IMRT-346: Feature/graylog env filter

### PR DESCRIPTION
**Pull request by jeffjohnson9046**
[IMRT-346](https://jira.fairwaytech.com/browse/IMRT-346):  
* Add documentation on how to apply an environment-specific filter in the Graylog UI 
* Add `fluentd-gelf-logging.md` file to collection of sample k8s deployment files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,8 +16,9 @@ IMRT is a suite of applications. These documents provide additional information
 6. [Release Notes](docs/release_notes.md)
 
 ### Common Tasks
-* [Delete an Item](docs/delete-item.md) 
+* [Delete an Item](docs/delete-item.md)
```
Comment (jeffjohnson9046) [inline README.md]: Formatting change?  Looks like the editor I chose removed a character here

File: docs/Deployment.AWS.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ If you are upgrading an existing system please first refer to the [Release Notes
     --dns-zone=[name of Route53 zone] \
     --ssh-public-key="[public key]" \
     --state=s3://[state store] \
-    --authorization=AlwaysAllow 
+    --authorization=AlwaysAllow
```
Comment (jeffjohnson9046) [inline docs/Deployment.AWS.md]: Formatting change?  Looks like the editor I chose removed a character here

File: docs/Deployment.AWS.md [hunk 3]
```diff
@@ -50,13 +50,13 @@ If you are upgrading an existing system please first refer to the [Release Notes
     --dns-zone=example.org \
     --ssh-public-key="~/.ssh/imrt-admin.pub" \
     --state=s3://kops-imrt-dev-state-store \
-    --authorization=AlwaysAllow 
-  
+    --authorization=AlwaysAllow
+
```
Comment (jeffjohnson9046) [inline docs/Deployment.AWS.md]: Formatting change?  Looks like the editor I chose removed a character here

File: docs/Deployment.AWS.md [hunk 4]
```diff
@@ -50,13 +50,13 @@ If you are upgrading an existing system please first refer to the [Release Notes
...
 ```
 
-* Edit the cluster configuration and change any desired settings, paying particular attention to the number of nodes, instance type and the size of the EC2 instances. For dev there are 2 "nodes" and one "master". You can find some documentation [here](https://github.com/kubernetes/kops/blob/master/docs/instance_groups.md).  
+* Edit the cluster configuration and change any desired settings, paying particular attention to the number of nodes, instance type and the size of the EC2 instances. For dev there are 2 "nodes" and one "master". You can find some documentation [here](https://github.com/kubernetes/kops/blob/master/docs/instance_groups.md).
```
Comment (jeffjohnson9046) [inline docs/Deployment.AWS.md]: Formatting change?  Looks like the editor I chose removed a character here

File: docs/Deployment.AWS.md [hunk 5]
```diff
@@ -50,13 +50,13 @@ If you are upgrading an existing system please first refer to the [Release Notes
...
 
->_**IMPORTANT:  When choosing a node size, avoid using `m3` AWS instance types.**  During deployments to AWS, using `m3.medium` AWS EC2 instances resulted in unpredictable behavior.  Additionally, the `m3.medium` instance type was not available for use in some AWS availability zones.  Consult AWS documentation for details on instance type availability._  
+>_**IMPORTANT:  When choosing a node size, avoid using `m3` AWS instance types.**  During deployments to AWS, using `m3.medium` AWS EC2 instances resulted in unpredictable behavior.  Additionally, the `m3.medium` instance type was not available for use in some AWS availability zones.  Consult AWS documentation for details on instance type availability._
```
Comment (jeffjohnson9046) [inline docs/Deployment.AWS.md]: Formatting change?  Looks like the editor I chose removed a character here

File: docs/Deployment.AWS.md [hunk 6]
```diff
@@ -68,7 +68,7 @@ If you are upgrading an existing system please first refer to the [Release Notes
    kops edit ig master-us-east-2b --name dev.imrt.org --state s3://kops-imrt-dev-state-store
    kops edit ig nodes --name dev.imrt.example.org --state s3://kops-imrt-dev-state-store
    </pre>
-   
+
```
Comment (jeffjohnson9046) [inline docs/Deployment.AWS.md]: Formatting change?  Looks like the editor I chose removed a character here

File: docs/Deployment.AWS.md [hunk 7]
```diff
@@ -110,7 +110,7 @@ The database that supports IMRT will be an Amazon Aurora Cluster using the Postg
 * Choose a database port (default is 5432)
 * Choose a **DB Parameter Group** appropriate for the chosen database engine (e.g. `default.aurora-postgresql9.6`)
 * Choose a **DB Cluster Parameter Group** appropriate for the chosen database cluster (e.g. `default.aurora-postgresql9.6`)
-* Decide whether to enable encryption based on your institution's policies 
+* Decide whether to enable encryption based on your institution's policies
```
Comment (jeffjohnson9046) [inline docs/Deployment.AWS.md]: Formatting change?  Looks like the editor I chose removed a character here

File: docs/Deployment.AWS.md [hunk 8]
```diff
@@ -206,8 +206,8 @@ Graylog will be installed in AWS following the directions [here](http://docs.gra
     * You will be prompted to supply an ssh key, you can select the same one used for creating the k8s cluster from the dropdown.
     * Click on the EC2 instance, and give it a name, for example `imrt-graylog-dev`
 * When the EC2 Instance comes up, ssh into it using the username `ubuntu` and the key you provided above.
-* Configure the server as described [here](http://docs.graylog.org/en/2.4/pages/installation/aws.html). 
-  * Stick to `http` for now, don't configure `https` 
+* Configure the server as described [here](http://docs.graylog.org/en/2.4/pages/installation/aws.html).
+  * Stick to `http` for now, don't configure `https`
```
Comment (jeffjohnson9046) [inline docs/Deployment.AWS.md]: Formatting change?  Looks like the editor I chose removed a character here

File: docs/Deployment.AWS.md [hunk 9]
```diff
@@ -274,7 +274,7 @@ If the access token is properly configured, the curl request will return a paylo
    * Edit the `configuration-service.yml` file locally and set the `GIT_PASSWORD`, and `ENCRYPT_KEY` values. _**NOTE:**_ _Do not check in these credentials!_
    * Create the configuration service
    <pre>kubectl apply -f configuration-service.yml</pre>
-   * Edit the `rabbit-cluster.yml` file and set the following: 
+   * Edit the `rabbit-cluster.yml` file and set the following:
```
Comment (jeffjohnson9046) [inline docs/Deployment.AWS.md]: Formatting change?  Looks like the editor I chose removed a character here

File: docs/Deployment.AWS.md [hunk 10]
```diff
@@ -284,15 +284,15 @@ If the access token is properly configured, the curl request will return a paylo
    kubectl apply -f rabbitmq_rbac.yml
    kubectl apply -f rabbit-cluster.yml
    </pre>
-   * Wait until the rabbit pods are up, you can monitor them using `kubectl get po`. 
+   * Wait until the rabbit pods are up, you can monitor them using `kubectl get po`.
```
Comment (jeffjohnson9046) [inline docs/Deployment.AWS.md]: Formatting change?  Looks like the editor I chose removed a character here

File: docs/Deployment.AWS.md [hunk 11]
```diff
@@ -284,15 +284,15 @@ If the access token is properly configured, the curl request will return a paylo
...
    * The value for `itembank.accessToken` in the `ap-imrt-iis.yml` file on the selected branch in the config repo must match the IAT Gitlab instance you are using, and must be encrypted with the `ENCRYPT_KEY` set for the configuration service. You can encrpyt a value on the configuration service using `kubectl` like this (use `kubectl get pods` to get the pod name):
    <pre>
    kubectl exec configuration-deployment-778dbb5675-pcwc7 -- curl http://localhost:8888/encrypt -d 'Test'
    </pre>
-   * The value for `spring.rabbitmq.password` in the `ap-imrt-iis.yml` file on the selected branch in the config repo must match the rabbit password you set above, and must be encrypted with the `ENCRYPT_KEY` set for the configuration service. You can encrpyt a value on the configuration service as described above.  
+   * The value for `spring.rabbitmq.password` in the `ap-imrt-iis.yml` file on the selected branch in the config repo must match the rabbit password you set above, and must be encrypted with the `ENCRYPT_KEY` set for the configuration service. You can encrpyt a value on the configuration service as described above.
```
Comment (jeffjohnson9046) [inline docs/Deployment.AWS.md]: Formatting change?  Looks like the editor I chose removed a character here

File: docs/Deployment.AWS.md [hunk 12]
```diff
@@ -365,18 +379,18 @@ To ensure IMRT is up-to-date with the content stored in source control, the Item
             restartPolicy: Never
   ```
 * Update the `schedule` value to match a time consistent with your institution's policies
-  * The schedule is configured using [Cron format](https://en.wikipedia.org/wiki/Cron#Overview) 
+  * The schedule is configured using [Cron format](https://en.wikipedia.org/wiki/Cron#Overview)
```
Comment (jeffjohnson9046) [inline docs/Deployment.AWS.md]: Formatting change?  Looks like the editor I chose removed a character here

File: docs/Deployment.AWS.md [hunk 13]
```diff
@@ -365,18 +379,18 @@ To ensure IMRT is up-to-date with the content stored in source control, the Item
...
   * Time is in UTC
   * To minimize potential performance impact, the recommendation is to schedule the cron to execute once daily after production hours
   * Additional details about cron jobs in Kubernetes environments can be found [here](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/)
 * Update the `image` to match the Item Ingest Service (IIS) image that has been deployed
-  * To find the name of the IIS image: 
+  * To find the name of the IIS image:
```
Comment (jeffjohnson9046) [inline docs/Deployment.AWS.md]: Formatting change?  Looks like the editor I chose removed a character here

File: docs/Deployment.AWS.md [hunk 14]
```diff
@@ -365,18 +379,18 @@ To ensure IMRT is up-to-date with the content stored in source control, the Item
...
 
     ```
     # identify the name of the pod hosting IIS
     $ kubectl get po | grep iis
     ap-imrt-iis-deployment-6f876df74-49rfv      1/1       Running     0          2d
-  
+
```
Comment (jeffjohnson9046) [inline docs/Deployment.AWS.md]: Formatting change?  Looks like the editor I chose removed a character here

File: docs/Deployment.AWS.md [hunk 15]
```diff
@@ -418,7 +432,7 @@ Once IMRT has been fully deployed, the database should be synchronized with the
 #### Updating YML file
    * When changes are made to a YML file, they can be applied to the cluster: <pre>kubectl apply -f xxx-service.yml</pre>
 
-#### Updating SNAPSHOT image (no docker tag change) 
+#### Updating SNAPSHOT image (no docker tag change)
```
Comment (jeffjohnson9046) [inline docs/Deployment.AWS.md]: Formatting change?  Looks like the editor I chose removed a character here

**Comment by malaffoon** [comment]
Thanks Jeff, gonna steal me some of this.

File: README.md (hunk 1 above)
Comment (jeffjohnson9046) [reply README.md]: Yep - generally use either Sublime Text or sometimes even vi if the change is real quick.  On a previous commit this file had a _ton_ of formatting changes, but I rolled it back & tested the various editing tools I use when updating these files.  It looks like macdown might put in some characters that Sublime Text and/or vi remove.  

To make life easier, I'll only use macdown from now on when editing `.md` files.

**Pull request merged**